api/circulation.py

```diff
@@ -771,9 +771,22 @@ def sync_bookshelf(self, patron, pin):
             # This is a remote loan. Find or create the corresponding
             # local loan.
             pool = loan.license_pool(self._db)
-            start = loan.start_date or now
```

Comment:
If there is no local loan and the remote has no opinion about the start date, the start date will be set to `now` inside `loan_to()`, so this isn't necessary. If there is a local loan and the remote has no opinion about the start date, we want to leave the existing start date alone rather than update it to the current date.